client/src/services/pinata.ts

```diff
@@ -0,0 +1,56 @@
+const jwt = `eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VySW5mb3JtYXRpb24iOnsiaWQiOiI0YmE2NDc0NS1hYWFmLTRiOWEtODE4YS1kZmIzODkxOWRlZWEiLCJlbWFpbCI6ImVuZ2luZWVyaW5nQGdpdGNvaW4uY28iLCJlbWFpbF92ZXJpZmllZCI6dHJ1ZSwicGluX3BvbGljeSI6eyJyZWdpb25zIjpbeyJpZCI6IkZSQTEiLCJkZXNpcmVkUmVwbGljYXRpb25Db3VudCI6MX0seyJpZCI6Ik5ZQzEiLCJkZXNpcmVkUmVwbGljYXRpb25Db3VudCI6MX1dLCJ2ZXJzaW9uIjoxfSwibWZhX2VuYWJsZWQiOmZhbHNlLCJzdGF0dXMiOiJBQ1RJVkUifSwiYXV0aGVudGljYXRpb25UeXBlIjoic2NvcGVkS2V5Iiwic2NvcGVkS2V5S2V5IjoiZjA5MzQwMTc2ZGQ2YjY3ZWRkMjQiLCJzY29wZWRLZXlTZWNyZXQiOiJmNzIyYzAyNjllZTdhM2Y0NzAwZDE2YTVmYjBmZGQ5OTZlMWNhMDM3Y2IwZDJkZmI1MmZiNWQyMjYwZjg1OTM3IiwiaWF0IjoxNjU2NTYyODkyfQ.qYcmSfln8YyHk7EtTWkvDZ4C5eiNBEprJj_gBgzupeI`;
```

Comment:
I feel like we should move this over to an env variable. I know its still possible to grab it from the client, but it would at least protect against github bots. wdyt?

Reply:
yes you are right! I'm going to do it and revoke the current token